addons/l10n_mx/models/chart_template.py

```diff
@@ -39,3 +39,9 @@ def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
             'show_on_dashboard': True,
         })
         return res
+
+    def _get_account_vals(self, company, account_template, code_acc, tax_template_ref):
```

Comment:
Please add a docstring, why you need this method, maybe Odoo could append this value in the original method.